judge/models/contest.py

```diff
@@ -218,6 +218,12 @@ def can_see_full_scoreboard(self, user):
             return True
         if not self.show_scoreboard:
             return False
+        if not self.is_visible:
```

Comment:
Code doesn't belong here (Does it even make sense to be here? Also, a lot of this code is already in `is_accessible_by`).